nixos/modules/system/boot/initrd-ssh.nix

```
@@ -33,8 +35,9 @@ in
     };
 
     shell = mkOption {
-      type = types.str;
-      default = "/bin/ash";
+      type = types.nullOr types.str;
+      default = null;
+      defaultText = ''"/bin/ash"'';
```

```suggestion
      defaultText = "/bin/ash";
```
I don't think we want the quotes in the final string.

---

Yes we do. The default text is supposed to be a valid expression for the option.